src/main/cpp/t06_longest.cpp

```diff
@@ -22,5 +22,21 @@
 using namespace std;
 
 int t06_longest() {
+    setlocale(LC_ALL,"Eng");
 
+    char str[256];
+    cin.getline(str,256);
+
+    char* ptr=strtok(str," .,!?");
+    char* high=ptr;
+
+    do {
+        if(strlen(high)<strlen(ptr))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Does not handle strings that are not \0-terminated; if given one it may perform an over-read (it could cause a crash if unprotected) (CWE-126).  ](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4605088)